src/components/Overlay.tsx

```diff
@@ -23,41 +23,28 @@ export interface OverlayProps {
   visible: boolean;
 }
 
-interface State {
-  fadeAnim: Animated.Value;
-  overlayStyle: StyleProp<ViewStyle>;
-}
+const Overlay: React.FC<React.PropsWithChildren<OverlayProps>> = ({ visible, children }) => {
+  const [fadeAnim] = React.useState(new Animated.Value(0));
+  const [overlayStyle, setOverlayStyle] = React.useState<StyleProp<ViewStyle>>(styles.emptyOverlay);
```

Comment:
What about importing the `useState` separately from React?

Reply:
> What about importing the `useState` separately from React? 

We can do that, of course. But I don't see that much difference imho. Will make the change when I'm back from holidays tho😉